control_panel_api/tests/test_views.py

```diff
@@ -4,19 +4,25 @@
 from rest_framework.test import APITestCase
 
 
-class UserViewTest(APITestCase):
+class AuthenticatedClientMixin(object):
     def setUp(self):
-        mommy.make('control_panel_api.User')
-        self.user = mommy.make('control_panel_api.User', is_staff=True)
-        self.client.force_authenticate(self.user)
+        super(AuthenticatedClientMixin, self).setUp()
+        self.client.force_login(mommy.make('control_panel_api.User'))
+
+
+class UserViewTest(AuthenticatedClientMixin, APITestCase):
+    def setUp(self):
+        super(UserViewTest, self).setUp()
```

Comment:
Is this `super().setUp()` necessary to call the `setUp()` in `AuthenticatedClientMixin`?